app/assets/stylesheets/application.scss

```diff
@@ -7,10 +7,7 @@
 @import "bootstrap/scss/bootstrap";
 @import "config/setup";
 @import "aos/dist/aos.css";
-@import "font-awesome";
-@import "star-rating.js/dist/star-rating.css";
-@import "swiper/swiper-bundle";
-@import "@mapbox/mapbox-gl-geocoder/dist/mapbox-gl-geocoder";
+// @import "swiper/swiper-bundle";
```

Comment:
Can you tell me why you commented this ?

Reply:
it breaks everything

Reply:
nevermind, now its working